test/baseTest.php

```diff
@@ -106,7 +106,7 @@ public function providerGetLangAndTranslationFile ()
         return array (
             array ("en", "en"),
             array ("fr,fr-fr;q=0.8,en-us;q=0.5,en;q=0.3", "fr"),
-            array ("pt-br,en-us;q=0.7,en;q=0.3", "en"),
+            array ("pt,en-us;q=0.7,en;q=0.3", "en"),
```

Comment:
Yes it's a failsafe,

I was thinking of Internet Explorer (unfortunately again ...) which  has the following Accept Language for my case (French) : 

`fr-FR`

With current code COPS is in english -> I'll have to fix this.

I think there other corner cases I'll have to think about.